### PR TITLE
open-cluster-management/backlog#10221 ansibleapi

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: apis/AnsibleJob.json.adoc [hunk 1]
```diff
@@ -0,0 +1,93 @@
+[#AnsibleJob-api]
+= AnsibleJob API (v1beta1)
+
+[[_rhacm-docs_apis_AnsibleJob_jsonoverview]]
+== Overview
+This documentation is for the AnsibleJob resource for {product-title}. Clusterset resource has four possible requests: create, query, delete and update.
```
Comment (jnpacker) [inline apis/AnsibleJob.json.adoc]: Clusterset = AnsibleJob

File: apis/AnsibleJob.json.adoc [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+=== URI scheme
+[%hardbreaks]
+__BasePath__ : /kubernetes/apis
+__Schemes__ : HTTPS
+
+
+=== Tags
+
+* cluster.open-cluster-management.io : Create and manage Clustersets
```
Comment (jnpacker) [inline apis/AnsibleJob.json.adoc]: AnsibleJobs, not clustersets

**Review by jnpacker — CHANGES_REQUESTED** [review]
Clusterset is referenced twice in this document, which is incorrect

**Review by jnpacker — APPROVED** [review]
Is there also a definition in the API for defining an instance of the CR? @cadawson

**Comment by cadawson** [comment]
There is not, but we can look at adding it.

**Comment by jnpacker** [comment]
Do we show the CR's for any of our CRD's?  I'm getting to the fact that we should.

**Comment by swopebe** [comment]
poke, @cadawson (looking at date opened, odd to have a PR open for this length of time, please let me know if you need help)

**Comment by cadawson** [comment]
Need to rework, so closing.

**Pull request closed without merging**